components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/GatewayPolicyData.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com)
```
Lets fix the same in other places as well.

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/12201/commits/1136a795360198388c6b4bd97029c20b0f898832

Thanks and Regards